src/lib.rs

```diff
@@ -797,16 +799,24 @@ pub enum BindingResource<'a> {
 }
 
 /// Operation to perform to the output attachment at the start of a renderpass.
-#[derive(Clone, Copy, Debug, Hash, PartialEq)]
+#[derive(Copy, Clone, Debug, Hash, Eq, PartialEq)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
```

Comment:
it seems to me that we should make it so `trace` feature derives serialization, and `replay` feature derives deserialization for these types. So `trace` and `replay` should both include `serde`